commcare/src/main/resources/webapp/js/directives.js

```diff
@@ -427,11 +427,10 @@
 
                 if (scope.$parent.selectedConfig) {
                     params.url = '../commcare/caseList/' + scope.$parent.selectedConfig.name + '?caseName=&dateModifiedStart=&dateModifiedEnd=';
-                }
-
-                if (scope.$parent.selectedConfig !== undefined) {
                     scope.downloadingCases = true;
-                    scope.$apply();
+                    setTimeout(function () {
```

Comment:
Also, since we are on 1.2.X angular now, isn't the best way to this according to the article to use $evalAsync (I'm guessing call elem.jqgrid in $evalAsync)